package.json

```diff
@@ -38,7 +38,7 @@
     "lint:fix": "nx affected --target=lint --fix",
     "new": "plop",
     "nx": "nx",
-    "postinstall": "is-ci || husky install",
+    "prepare": "is-ci || husky install",
```

Comment:
Did this change again? I thought on yarn 2+ we were supposed to use postinstall: https://typicode.github.io/husky/getting-started.html#yarn-2

Reply:
I missed that note in the docs. I'll switch it back